iocore/net/quic/Makefile.am

```diff
@@ -116,7 +117,8 @@ check_PROGRAMS = \
   test_QUICTypeUtil \
   test_QUICVersionNegotiator \
   test_QUICFrameRetransmitter \
-  test_QUICAddrVerifyState
+  test_QUICAddrVerifyState \
+  test_QUICStreamFactory
```

Comment:
I might be missing something, where is source file of this check program?

Reply:
Ops.. missing that.

Reply:
Fixed.